### ISSUE TITLE
Remember the patch files loaded

### ISSUE THREAD

**Issue by ensonic** [Kymorphia/alkimiya#2]
When starting Alkimiya auto load all patch files one had previously loaded.

**Comment by Kymorphia** [comment]
Good idea. Instrument library support is planned for cataloging and searching through many instrument files, both locally and remotely, but having the last loaded instruments would also be good to have. We'll try and add this in the next 0.9.1 release.
